src/Tempest/Generation/src/StubFileGenerator.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Generation;
+
+use Tempest\Console\Console;
+use Tempest\Generation\Exceptions\StubFileGenerationFailedException;
+use Tempest\Support\PathHelper;
+use Tempest\Support\StringHelper;
+use Throwable;
+
+/**
+ * This class can generate a file from a stub file with additional useful methods.
+ * It only works with PHP class files.
```

Comment:
This worries me, what if we want a `make:view-component --anonymous` command?

Reply:
I'm not ultra familiar with the `nette/php-generator` package but in case we use the `ClassManipulator` to create the stub and manipulate it.
So currently I don't think it's possible to create a stub from another file of php classes.
If we wanted to, I think something needs to be refactor like using a `FileManipulator` or something like this.

Do you think this could comes with the PR that will add the `make:view-component` command ?
I think it could be the right scope, doing things/refactor when we need it, and then update this PHPDoc

Reply:
So, now this is possible with the refactoring of the `StubFileGenerator` to support `PublishesFiles` features